lib/src/main/java/com/github/kevinsawicki/http/HttpRequest.java

```diff
@@ -3178,6 +3178,25 @@ public HttpRequest form(final Map<?, ?> values, final String charset)
   }
 
   /**
+   * Write the json string
+   *
+   * @param jsonString
+   * @return this request
+   * @throws HttpRequestException
+   */
+  public HttpRequest json(final String jsonString)
+      throws HttpRequestException {
+    contentType(CONTENT_TYPE_JSON);
+    try {
+      openOutput();
```

Comment:
You could just call `send` here instead which would make this method a little simpler:

```java
contentType(CONTENT_TYPE_JSON);
send(jsonString);
```